### ISSUE TITLE
nexmo-java not support maven?

### ISSUE THREAD

**Issue by youngchen1987** [Nexmo/nexmo-java#37]
nexmo-java not support maven?

**Comment by judy2k** [comment]
Hi @youngchen1987 

We do support Maven. Please try the following:

```xml
<dependency>
      <groupId>com.nexmo</groupId>
      <artifactId>client</artifactId>
      <version>2.0.0-SNAPSHOT</version>
</dependency>
```

If it doesn't work, let us know and we'll fix it!

**Comment by AdamGhani** [comment]
For me, I have to point to the repo.
`
<repositories>
    <repository>
            <id>nexmo</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
    </repository>
<repositories>
`

**Comment by judy2k** [comment]
Thank you @AdamGhani! I didn't have a chance to check whether that was required.

@youngchen1987 I recommend you add this repositories snippet as well as the dependency snippet. Let us know how you get on!

**Comment by judy2k** [comment]
I will validate this feedback and add it to the README.

**Comment by stepio** [comment]
Any plans to do a release version and upload it to maven central?

**Comment by judy2k** [comment]
Absolutely! It's coming as soon as the API is fully stabilized. We have to
move some objects around before we're happy for a final production release.

**Comment by judy2k** [comment]
We've released a final version of 2.0.0, and how to use it with Maven is documented in the README, so I'm closing this issue.
